javascript/javascript3/week3/promises.js

```diff
@@ -0,0 +1,52 @@
+//soujanyapolavarapu  sandhana14   cph-kiwi
+/* 
+Fetch all the 3 classmates repositories at the same time using Promise.all. Remember the all at once
+ exercise here?
+
+When you have the data for the different repositories, render the fullname of the repo, url of the repo, 
+and the owner of the repo.
+ See github-repos as an example of how the renderered repos should look. You are more than welcome to style 
+ it a bit nicer! */
+const url1 = "https://api.github.com/search/repositories?q=user:soujanyapolavarapu";
+const url2 = "https://api.github.com/search/repositories?q=user:qamarfj";
+const url3 = "https://api.github.com/search/repositories?q=user:cph-kiwi";
+
+
+class GetClass {
+    constructor(url) {
+        this.url = url;
+
+    }
+    get() {
+        fetch(this.url)
+            .then(response => response.json())
+            .then((studentData) => {
+                const ulElement0 = document.createElement('ul');
+                const liElement0 = document.createElement('li');
+                liElement0.innerHTML = '<h2>' + studentData.items[0].owner.login + ' repositories<\h2>';
+                //console.log('\n\n'+studentData.items[0].owner.login +' repositries \n\n');
+                const ulElement1 = document.createElement('ul');
+                studentData.items.filter(element => {
+
```

Comment:
I would suggest using forEach (unfortunately I also didn`t use it in my hw) here instead of filter, because we don't have a condition to filter the array here, we have to change every element of the array.

Reply:
done